config.m4

```diff
@@ -469,9 +485,9 @@ Build configuration:
   Extra CFLAGS                                     : $STD_CFLAGS $EXTRA_CFLAGS
   Developers flags (slow)                          : $MAINTAINER_CFLAGS
   Code Coverage flags (extra slow)                 : $COVERAGE_CFLAGS
-  System mongoc                                    : $PHP_LIBMONGOC
-  System libbson                                   : $PHP_LIBBSON
-  System libmongocrypt                             : $PHP_LIBBSON
+  System mongoc                                    : $PHP_SYSTEM_LIBS
```

Comment:
We might as well rewrite this as "System libmongoc" while we're changing this line.

Reply:
Done.